oem/ibm/libpldmresponder/file_io_type_dump.hpp

```diff
@@ -60,8 +60,18 @@ class DumpHandler : public FileHandler
     {}
 
   private:
-    static int fd;     //!< fd to manage the dump offload to bmc
-    uint16_t dumpType; //!< type of the dump
+    static int fd;              //!< fd to manage the dump offload to bmc
+    uint16_t dumpType;          //!< type of the dump
+    std::string resDumpDirPath; //!< directory where dump entry is stored
```

Comment:
Can you please change this name to to something like resDumpRequestDirPath and update the comment like "directory where the resource dump request parameter file is stored"